pkg/services/govmomi/power.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package govmomi
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha1"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/types"                         //nolint
```

Comment:
Q: Why are we using nolint here on imports?

Reply:
I'm getting this strange error
`File is not `gci`-ed with --skip-generated -s standard,default (gci)` and running the linter locally.
Not sure what attributed.

Reply:
Format with https://github.com/daixiang0/gci

Reply:
@laozc Can you run `make fix` or `make lint-fix` (depending if you recently rebased onto main)

Reply:
Is done, can be resolved